### PR TITLE
Michael

### PR DESCRIPTION
**Pull request by SongeDaiDai**
All updates made.

Still need to modify the Title and Gif :)

### CODE REVIEW

File: src/main/webapp/BookForm.jsp [hunk 1]
```diff
@@ -1,58 +1,75 @@
...
+                <div class="subtitle">Publisher</div>
+                <select class="select" id="publisher" name="publisher" multiple>
+                    <option disabled selected>-- select an option --</option>
+                    <option class="option">John Wiley &amp; Sons</option>
+                    <option class="option">BigNerd Ranch</option>
+                    <option class="option">Industrial Press</option>
+                    <option class="option">Armond Dalton</option>
+                </select>
+                <div class="cut"></div>
+                <label for="publisher" class="placeholde"></label>
```
Comment (NickeeCoco) [inline src/main/webapp/BookForm.jsp]: Little typo here, should be `class="placeholde**r**"`

File: src/main/webapp/BookForm.jsp [hunk 2]
```diff
@@ -1,58 +1,75 @@
...
+            <div class="input-container">
+                <div class="subtitle">Subject</div>
+                <label><input  type="checkbox" name="subject" value="classic">Classic</label>
+                <label><input  type="checkbox" name="subject" value="fantasy">Fantasy</label>
+                <label><input  type="checkbox" name="subject" value="history">History</label>
+                <label><input  type="checkbox" name="subject" value="horror">Horror</label>
+            </div>
+            <input class="submit" type="submit" value="Add Book" />
+            <p>
+                <img src="cat-reading.gif" width="50%" style="padding-left: 25%; border-radius: 5%;">
```
Comment (NickeeCoco) [inline src/main/webapp/BookForm.jsp:69]: It's cute, but will the teacher like it? ≡(▔﹏▔)≡

Comment (SongeDaiDai) [reply src/main/webapp/BookForm.jsp:69]: Well, it's fitting with the assigment :)
And for real, if he can't handle a bit of "fun" that's sad.

File: src/main/webapp/BookForm.jsp (hunk 1 above)
Comment (SongeDaiDai) [reply src/main/webapp/BookForm.jsp]: Corrected will push in a moment :)

**Review by SongeDaiDai — COMMENTED** [review]
Added the R to Placeholder line 50 of Form

**Pull request merged**